src/constants/ecosystem.ts

```diff
@@ -14,6 +14,7 @@ export const DIVERGENT_CATEGORY_MAP = {
   Social: ["Social"],
   Tooling: ["Tooling"],
   Wallet: ["Wallet", "Hardware Wallet"],
+  Bridge: ["Bridge"],
```

Comment:
For data consistency, please delete the **Bridge** in the **Infra** group.

Reply:
Removed